docs/reference/modules/apostrophe-images/README.md

```diff
@@ -11,6 +11,23 @@ a website, and to manage a library of them for reuse.
 Each `apostrophe-image` doc has an `attachment` schema field, implemented
 by the [apostrophe-attachments](/reference/modules/apostrophe-attachments) module.
 
+::: warning NOTE
+By default, `apostrophe-images-widgets` take the `title` field of the image piece to fill
```

Comment:
Spell out "image" or use backticks to make it inline code.

Reply:
Modified in core